library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -108,7 +113,18 @@ public void onCreate(@NonNull SQLiteDatabase db) {
      */
     @Override
     public void onUpgrade(@NonNull SQLiteDatabase db, int oldVersion, final int newVersion) {
-        // no upgrade path yet
+        if (oldVersion == 1 && newVersion == 2) {
+            upgradeFromVersionOneToVersionTwo(db);
+        }
```

Comment:
I've up the DB_VERSION to 2 and this is how the migration is done. 
Shout if you see something is missing here since this is quite critical